src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -234,6 +235,15 @@ protected void submit(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
         }
     }
 
+    @Override
+    protected @Nonnull String getSuccessfulDestination() {
+        FolderComputation<I> computation = getComputation();
+        if (computation != null) {
```

Comment:
@jglick I'll remove the `if` after this https://github.com/jenkinsci/cloudbees-folder-plugin/pull/60 is merged.